README.md

```diff
@@ -30,7 +28,8 @@ cd bitshares-ui
 Before launching the GUI you will need to install the npm packages:
 
 ```
-npm install
+npm ci
```

Comment:
ci is recommended to use for production builds. For development do `npm install`.

In both cases `install-charting-library` is called from `postinstall` script.

Reply:
This was tested over *Ubuntu 20.04.2 LTS (GNU/Linux 5.4.0-73-generic x86_64)*:

*In both cases *install-charting-library* isn't getting installed during postinstall script with a warning says:*

npm WARN lifecycle BitShares2-light@5.0.20210720-rc1~postinstall: cannot run in wd BitShares2-light@5.0.20210720-rc1 npm run install-charting-library (wd=/root/bitshares-ui)

The above will prevent the build from completing later with an error:

ERROR in ./app/components/Exchange/TradingViewPriceChart.jsx
Module not found: Error: Can't resolve '../../../charting_library/charting_library.esm' in '/root/bitshares-ui/app/components/Exchange'
 @ ./app/components/Exchange/TradingViewPriceChart.jsx 10:18-75
 @ ./app/components/Exchange/Exchange.jsx
 @ ./app/components/Exchange/ExchangeContainer.jsx
 @ ./app/App.jsx
 @ ./app/AppInit.jsx
 @ ./app/index.js
 @ ./app/Main.js

Reply:
Thanks for the details. I guess it's better to fix the `postinstall` script?

Reply:
According to this link: https://stackoverflow.com/questions/18136746/npm-install-failed-with-cannot-run-in-wd , I guess you got the error because you were installing with `root`?

Reply:
@abitmore thanks, yes was installing using root user on a fresh installed server; I agree it's not a good practice to dvelop under root, perhaps writing a note, or changing the procedure is helpful in this case as users are facing the issue and last time was reported yesterday on telegram, perhaps generalizing a single commend such as *npm install --unsafe-perm* is a solution but maybe "unsafe" looks ugly, more suggestions are appreciated.

Reply:
Yes it does look ugly. :P I think it's OK to add a note about root though. @sschiessl-bcp any idea?

Reply:
I'd say best would be to add a note to do this with a non-root user